### PR TITLE
add navbar and routes

### PR DESCRIPTION
**Pull request by IgnasKia**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { FormComponent } from './components/form/form.component';
+import { HomeComponent } from './components/home/home.component';
+export { Routes } from '@angular/router';
 
-const routes: Routes = [];
+// add this
```
Comment (PovilasKM) [inline src/app/app-routing.module.ts]: Doesn't seem like a useful comment.

**Review by ignaspangonis — CHANGES_REQUESTED** [review]
needs refactoring (material folder)

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { FormComponent } from './components/form/form.component';
+import { HomeComponent } from './components/home/home.component';
+export { Routes } from '@angular/router';
```
Comment (PovilasKM) [inline src/app/app-routing.module.ts:5]: Try to use some kind of plugin to auto sort imports when you are savings a file.

File: src/app/components/form/form.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+interface Mentor {
+  value: string;
```
Comment (PovilasKM) [inline src/app/components/form/form.component.ts]: What's the difference between value and viewValue? Perhaps fields could be renamed to reflect it better?

File: src/app/components/form/form.component.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+@Component({
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.scss']
+})
+export class FormComponent implements OnInit {
+  
+  mentor: Mentor[] = [
+    {value: 'Minijus Laukaitis', viewValue: 'Minijus Laukaitis'},
```
Comment (PovilasKM) [inline src/app/components/form/form.component.ts]: Try not to use real names.

File: src/app/components/form/form.component.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    {value: 'Darius Pazusinskis', viewValue: 'Darius Pazusinskis'},
+    {value: 'Mantas Boronilščikovas', viewValue: 'Mantas Boronilščikovas'}
+  ];
+
+  constructor() { }
```
Comment (PovilasKM) [inline src/app/components/form/form.component.ts]: Don't leave empty functions. Add them back when they are needed.

**Comment by PovilasKM** [comment]
Just a general question. Are you using some kind of formatter? For example prettier.

File: src/app/components/form/form.component.ts (hunk 3 above)
Comment (IgnasKia) [reply src/app/components/form/form.component.ts]: I don't know, I just started to learn Angular Material and used example 
image

File: src/app/components/form/form.component.ts (hunk 4 above)
Comment (IgnasKia) [reply src/app/components/form/form.component.ts]: Good point

**Comment by IgnasKia** [comment]
Not yet, but I'll take an interest into formatter

File: src/app/components/form/form.component.ts (hunk 3 above)
Comment (IgnasKia) [reply src/app/components/form/form.component.ts]: but this is just for demo version now

**Review by ignaspangonis — APPROVED** [review]

**Pull request merged**